### PR TITLE
Np 16240 total funding amount

### PR DESCRIPTION
**Pull request by torarnet**

### CODE REVIEW

File: cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/CristinFundingAmount.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    @JsonCreator
+    public CristinFundingAmount(@JsonProperty(CRISTIN_CURRENCY_CODE) String currencyCode,
+                                @JsonProperty(AMOUNT) Double amount) {
+        this.currencyCode = currencyCode;
+        this.amount = amount;
+    }
+
+    public String getCurrencyCode() {
+        return currencyCode;
```
Comment (brinxmat) [inline cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/CristinFundingAmount.java:28]: Is the currency code an ISO4217 code?

File: cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/FundingAmount.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    @JsonProperty(AMOUNT)
+    private final transient Double amount;
+
+    @JsonCreator
+    public FundingAmount(@JsonProperty(CURRENCY_CODE) String currencyCode, @JsonProperty(AMOUNT) Double amount) {
+        this.currencyCode = currencyCode;
+        this.amount = amount;
+    }
+
+    public String getCurrencyCode() {
```
Comment (brinxmat) [inline cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/FundingAmount.java:28]: Probably just "currency"

File: cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/FundingAmount.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        return currencyCode;
+    }
+
+    public Double getAmount() {
```
Comment (brinxmat) [inline cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/FundingAmount.java:32]: probably "value"

**Review by brinxmat — DISMISSED** [review]

File: cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/CristinFundingAmount.java (hunk 1 above)
Comment (torarnet) [reply cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/CristinFundingAmount.java:28]: The string "NOK" is hardcoded in the Cristin model. We have to change that when we add support for creation and update of funding amount

**Review by brinxmat — APPROVED** [review]

**Pull request merged**